tools/extra/pac/fpgabist/bist_dma.py

```diff
@@ -55,26 +55,33 @@ def run_cmd(self, cmd):
             ret = 1
         return ret
 
-    def run(self, gbs_path, bus_num, bd_id=0, guid=''):
+    def run(self, gbs_path, bdf, bd_id=0, guid=''):
         if gbs_path:
-            bc.load_gbs(gbs_path, bus_num)
+            bc.load_gbs(gbs_path, bdf)
         ret = 0
         for func, param in self.executables.items():
             if bd_id in dma_list:
                 for i, c in dma_list[bd_id].items():
                     name, size = c
                     if self.dev_id != int(param, 16):
                         continue
-                    cmd = "{} {} -B 0x{} -D {} -S {}".format(func, param,
```

Comment:
Is this working? Removing the leading `0x` before this was required so are you passing the leading `0x` with each element of the bdf?

Reply:
No, DMA is broken still. I need the changes that were cherry picked into the internal master to test it, so was waiting for this pull to go in.